contrib/dockerd-rootless.sh

```diff
@@ -15,6 +15,7 @@
 # * DOCKERD_ROOTLESS_ROOTLESSKIT_PORT_DRIVER=(builtin|slirp4netns|implicit): the rootlesskit port driver. Defaults to "builtin".
 # * DOCKERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SANDBOX=(auto|true|false): whether to protect slirp4netns with a dedicated mount namespace. Defaults to "auto".
 # * DOCKERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SECCOMP=(auto|true|false): whether to protect slirp4netns with seccomp. Defaults to "auto".
+# * DOCKERD_ROOTLESS_ROOTLESSKIT_DISABLE_HOST_LOOPBACK=(true|false): disallow to enable host loopback and use 10.0.2.2 (in the case of slirp4netns) to connect to the host loopback. Defaults to "true".
```

Comment:
A suggestion on the wording, to align with rootlesskit/slirp4netns docs - which seem to be specific about `127.0.0.1` (rather than `127.0.0.0/8` or `::1`) ... but please check I've understood the option properly!

 ```suggestion
# * DOCKERD_ROOTLESS_ROOTLESSKIT_DISABLE_HOST_LOOPBACK=(true|false): prohibit connections to 127.0.0.1 on the host (including via 10.0.2.2, in the case of slirp4netns). Defaults to "true".
```

Reply:
Yes, your suggestion are correct, I have added it.

Reply:
@robmry Yes, your suggestion are correct, I have added it.